packages/caravan-fees/src/btcTransactionComponents.ts

```diff
@@ -0,0 +1,231 @@
+import { Satoshis, BTC, TxOutputType } from "./types";
+import { satoshisToBTC } from "./utils";
+import BigNumber from "bignumber.js";
+
+/**
+ * Abstract base class for Bitcoin transaction components.
```

Comment:
```suggestion
 * Abstract base class for Bitcoin transaction components (inputs and outputs).
```
It's not immediately obvious what a "transaction component" is.

Reply:
[done](https://github.com/caravan-bitcoin/caravan/pull/114/commits/6662b2d56bc5f1cef57c403cdc13c2a0ccaac980#diff-db5fc022ab08b92fb2a606342674432f918748c762c9fcb907bc43d3531d512bR6)